### ISSUE TITLE
The keyboard focus ring is invisible because it's the same colour as the buttons

### ISSUE THREAD

**Issue by seanh** [hypothesis/lms#2296]
The focus ring that's shown around the focused button is black but many of our buttons are also black, so the focus ring is nearly invisible:

![Peek 2020-12-11 13-46](https://user-images.githubusercontent.com/22498/101910774-553c4800-3bb7-11eb-8dcb-bc935defa9d8.gif)

![Peek 2020-12-11 13-48](https://user-images.githubusercontent.com/22498/101910939-93396c00-3bb7-11eb-9301-8a0b323b224e.gif)

**Comment by robertknight** [comment]
This looks like the default focus ring color, which depends on the browser. For the vast majority of users it will either be Chrome's dark blue (a little bit better, but still quite hard to see):

<img width="583" alt="Chrome focus ring" src="https://user-images.githubusercontent.com/2458/101911249-07740f80-3bb8-11eb-9e13-8b1fb0f81224.png">

Or Safari's lighter blue (much better):

<img width="540" alt="Safari focus ring" src="https://user-images.githubusercontent.com/2458/101911390-34282700-3bb8-11eb-8022-c904c5b329d6.png">

Or for a minority of users, Firefox's one (terrible):

<img width="546" alt="Firefox focus ring" src="https://user-images.githubusercontent.com/2458/101911521-6043a800-3bb8-11eb-9ab9-dbbc7350d1ec.png">

Whatever we do here we'll need to make sure the client and LMS frontend are both in sync.

**Comment by seanh** [comment]
Can the focus ring colour be easily styled? If so we might want to style it so we can make sure it's a colour that doesn't clash with any of our buttons. If not, there might not be anything we can do about this?

**Comment by seanh** [comment]
FYI I'm just using Chrome on Linux in the above screenshots. I've tried Chrome and Chromium, and tried with different Chrome themes, it looks like Chrome always uses black for the focus ring on Linux (at least on Ubuntu / GNOME)

**Comment by robertknight** [comment]
> Can the focus ring colour be easily styled? 

We can replace the system focus ring entirely with something custom. I don't know how much ability we have to tweak the default focus ring (eg. change the color or thickness but keep the shape). We'll have to figure that out.

**Comment by LMS007** [comment]
Closing this out as most of the buttons have been converted to the new shared button/style including all the ones outlined in this ticket. The grader remains, but is not problicamic as it implemented focus correctly.
